frameworks/PHP/symfony/setup.sh

```diff
@@ -2,9 +2,8 @@
 
 fw_depends mysql php7 nginx
 
-sed -i 's|localhost|'"${DBHOST}"'|g' deploy/nginx.conf
```

Comment:
@javiereguiluz It's important to let this replacement to have the good DBHost deployed.

Reply:
I can't find `DBHOST` in the `deploy/nginx.conf` because they use an env var set by the server: https://github.com/TechEmpower/FrameworkBenchmarks/pull/3162/files#diff-aad0e0f301e725ec260300b248f1ba33R59  But I can be wrong because I don't know this well.

Reply:
In fact in the following fastcgi_param ``DATABASE_URL "mysql://benchmarkdbuser:benchmarkdbpass@localhost/hello_world";``, ``localhost`` must be replaced by the content of ``${DBHOST}``.